### PR TITLE
Use plain return type instead of named result parameter

### LINKED ISSUE #19: Named result parameters
@quii Named result parameters are recommended only in certain context. The last refactor in `Hello, World` gives an impression to use named results parameters in general to make the documentation clear. Should we update it to not use named result param since its pretty clear from the function name itself what it is going to return?
If we agree on this, I can raise the PR for same.

Ref: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

### PR DESCRIPTION
**Pull request by kunalkapadia**
Resolves #19

### CODE REVIEW

**Comment by kunalkapadia** [comment]
@quii Please go [this](https://github.com/quii/learn-go-with-tests/pull/21/files#diff-2ee7d41aa45b5470afc28f2f6775e5bdR48) specifically.

File: structs/readme.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Notice the new format string? The `f` is for our `float64` and the `.2` means pr
 ## Write the minimal amount of code for the test to run and check the failing test output
 
 ```go
-func Perimeter(width float64, height float64) (perimeter float64) {
+func Perimeter(width float64, height float64) float64 {
 	return
```
Comment (quii) [inline structs/readme.md]: This wont compile will it?

Comment (kunalkapadia) [reply structs/readme.md]: @quii It will compile right? Otherwise tests would have failed?

Comment (quii) [reply structs/readme.md]: You cant do naked returns without a named param

The markdown reads

```go
func Perimeter(width float64, height float64) float64 {
	return
}
```
Try it here: https://play.golang.org/p/bHx0k4AdZ0k

Even though the `./build.sh` is successful doesn't mean the markdown is right, as so far i dont know a way to compile the go code inside markdown sadly :(

Comment (kunalkapadia) [reply structs/readme.md]: @quii My bad, I overlooked it.
Also I meant I used the same code in one of the go files and it worked fine but it wan't the empty `Parimeter` function. Let me fix this asap.

**Comment by quii** [comment]
Nice work @kunalkapadia

**Comment by kunalkapadia** [comment]
@quii Thanks for those quick fixes in README!

**Comment by quii** [comment]
No worries! They're easy mistakes to make

I feel like for my own sanity's sake at some point i need to write something that parses markdown files with `go` snippets in it and make sure they compile 😱

**Comment by kunalkapadia** [comment]
@quii That really is a good idea and could be immensely useful for several other similar repos.

**Pull request merged**